shared/desktop/app/node.desktop.tsx

```diff
@@ -32,9 +32,9 @@ let saltpackFilePath: string | null = null
 const installCrashReporter = () => {
   if (env.KEYBASE_CRASH_REPORT) {
     console.log(`Adding crash reporting (local). Crash files located in ${Electron.app.getPath('temp')}`)
+    Electron.app.setPath('crashDumps', cacheRoot)
```

Comment:
Seems to have changed, relevant docs at https://www.electronjs.org/docs/api/crash-reporter